app/views/employees/_form.html.erb

```diff
@@ -91,62 +91,62 @@
 
 <div class="checkbox field">
     <%= f.check_box :has_food_hygiene, {class: "checkbox__input"} %>
-    <%= f.label :has_food_hygiene, "Does this person have food hygiene qualification?", class: "checkbox__label" %>
+    <%= f.label :has_food_hygiene, "Does this person have food hygiene qualification?", class: "checkbox__label" , min: Date.new(1920, 01, 01), max: Date.today%>
 </div>
 
 <div class="field">
     <%= f.label :food_hygiene_achieved_on, "If so, when did they achieve it?", class: "field__label" %>
-    <%= f.date_field :food_hygiene_achieved_on, class: "field__input" %>
+    <%= f.date_field :food_hygiene_achieved_on, class: "field__input", min: Date.new(1920, 01, 01), max: Date.today %>
```

Comment:
This is the validation, I add in the all date pickers, since they are about achievement. And I add the min date too